src/pages/MainPage.tsx

```diff
@@ -0,0 +1,22 @@
+import PageContainer from 'components/@commons/PageContainer';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { RootState } from 'states/store';
+
+import NoGroupSection from 'components/admin-home/NoGroupSection';
+import AdminHomeSection from 'components/admin-home/AdminHomeSection';
+import OnBoardingSection from 'components/OnBoardingSection';
+
+const MainPage = (): JSX.Element => {
+  const loginState = useSelector((state: RootState) => state.login);
+  const isLogin: boolean = loginState.islogin;
+  const isGroup: boolean = loginState.userData?.groupName.length > 0;
+
+  return (
+    <PageContainer withoutHeader={!isLogin} withoutBottonBar={!isGroup}>
+      {isLogin ? isGroup ? <AdminHomeSection /> : <NoGroupSection /> : <OnBoardingSection />}
```

Comment:
삼항 연산자 두개가 같이 쓰여서 해석하기 다소 복잡한 라인입니다!

```suggestion
      {isLogin && isGroup ? <AdminHomeSection /> : <NoGroupSection />}
      {!isLogin && < OnBoardingSection />}
```
이렇게만 바꿔도 좋을 것 같습니다!